### PR TITLE
Add a trophies section to the PR Overview page

### PR DESCRIPTION
**Pull request by kjohn1922**
### Justification
Add a trophies section to the right-hand pane on the Overview tab of a PR containing "Trophies" for notable PR numbers. This was a quirky feature of Review Board that I thought it would be fun to bring over to AzDo.
### Testing
Works for: Firefox and Chrome; light theme and dark theme; dev.azure.com/account and account.visualstudio.com.
Test PRs from NI: There are no valid milestone PRs that I could find, so I tested the logic by hard-coding a PR number. PRs 75057 and 70007 are valid for the fish trophy. 

### Known issues
Does not work for the "new PR experience" preview feature. I have version that partially works for the new PR view that I could finish out when that becomes the default view. This version does not cause any page or script errors for the new view - the code terminates early when it can't find the PR author from the same part of the HTML.

### Screenshots
Review Board version:
review-board-trophies
AzDo Userscript version:
milestone-trophy-large
fish-trophy

### CODE REVIEW

**Comment by kjohn1922** [comment]
Wasn't aware of the style checks or the build process. I'll fix those issues and re-post. This is my first excursion into both userscripts and github, so bear with me.

**Comment by kjohn1922** [comment]
> Wasn't aware of the style checks or the build process. I'll fix those issues and re-post. This is my first excursion into both userscripts and github, so bear with me.

The npm build no longer fails (didn't know to run it before). Learned some things about javascript variables and style guidelines. Apologies to all.

**Comment by alejandro5042** [comment]
LOL - This is amazing. Let me test this and get back to you today/tomorrow!

**Comment by wireddown** [comment]
🤩🤩🤩

👀👀👀

**Comment by gerardogalvez** [comment]
🏆🏆🏆

File: src/azdo-pr-dashboard.user.js [hunk 1]
```diff
@@ -272,6 +273,48 @@
...
+        .append(sectionContent);
+
+      // Milestone trophy: Awarded if pull request ID is greater than 1000 and is a non-zero digit followed by only zeroes (e.g. 1000, 5000, 10000).
+      if (pullRequestId >= 1000 && pullRequestId.match('^[1-9]0+$')) {
+        const milestoneTrophyMessage = $('<span>)').text(`${pullRequestAuthor} got pull request #${pullRequestId}!`);
+        sectionContent.append(milestoneTrophyMessage.prepend('&ensp;🏆&emsp;'));
+        trophyAwarded = true;
+      }
+
+      // Fish trophy: Give a man a fish, he'll waste hours trying to figure out why.
```
[the remark this answers is not in the document]
Comment (kjohn1922) [reply src/azdo-pr-dashboard.user.js]: Done!
I initially just copied the documentation for the trophy types from the review board original, which you can see here: [https://reviews.reviewboard.org/r/8324/diff/3/#index_header](https://reviews.reviewboard.org/r/8324/diff/3/#index_header)
They didn't have a comment to explain the trophy, I suppose to keep the mystery alive. 😆

**Pull request merged**